typescript-timers/interval.ts

```diff
@@ -0,0 +1,14 @@
+const title = document.querySelector('.countdown-display');
+
+if (title) {
+  let count = 4;
+  const countdownInterval = setInterval(() => {
+    if (count > 1) {
```

Comment:
We don't want to put these inside of `if` statements. What happens if our query fails? This doesn't run and don't receive an error message. We should be checking this like we have in previous exercises where we check if not the element, and instantiate a new error that we throw.
```if (!$title) throw new Error("issue querying for title")```
This way if we have an issue, we'll throw an error and know that it was with the title query